### PR TITLE
Fix empty grid column rendering

### PR DESCRIPTION
**Pull request by NickLennonLiu**
## Summary

- Treat grid columns without children as empty Markdown instead of passing `undefined` to `marked.parse`.
- Add a regression test for a grid containing an empty column.

## Root cause

`parseGridColumn` initialized `inner` as an empty string, but optional chaining reassigned it to `undefined` when `block.children` was missing. That `undefined` value was passed to `markdownToHTML`, which calls `marked.parse` and throws.

## Validation

- `COREPACK_ENABLE_AUTO_PIN=0 yarn workspace feishu-docx test`
- `COREPACK_ENABLE_AUTO_PIN=0 yarn workspace feishu-docx build`
- `COREPACK_ENABLE_AUTO_PIN=0 FEISHU_APP_ID=dummy FEISHU_APP_SECRET=dummy FEISHU_SPACE_ID=dummy yarn workspace feishu-pages test`

Note: plain `COREPACK_ENABLE_AUTO_PIN=0 yarn test` still requires the same feishu-pages test setup: build `feishu-docx` first and provide the required Feishu env vars.

### CODE REVIEW

File: feishu-docx/src/markdown_renderer.ts [hunk 1]
```diff
@@ -709,7 +709,7 @@ export class MarkdownRenderer extends Renderer {
           const child = this.blockMap[childId];
           return this.parseBlock(child, 0);
         })
-        .join('\n');
+        .join('\n') ?? '';
```
[the remark this answers is not in the document]
Comment (NickLennonLiu) [reply feishu-docx/src/markdown_renderer.ts]: Updated to default `block.children` to an empty array before mapping, which makes the missing-children case explicit and avoids relying on optional chaining behavior.
